### DISCUSSION TITLE
Inserimento collegamenti app esterne

### DISCUSSION THREAD

**Discussion by Lucaccc** [iisgiua/giuaschool#469] [Richieste di aiuto] [answered]
Buon pomeriggio, volevo sapere come potessi aggiungere dei collegamenti per le app nella sezione relativa della pagina di accesso (per capirci, dove si trovano i collegamenti per scaricare le app del registro).
Grazie.

**Comment by trinko** [comment, marked as answer]
Copia il template **templates/app/info.html.twig** nella cartella **PERSONAL/templates/app/info.html.twig**

Quindi modifica il nuovo file che hai inserito: 
- puoi aggiungere a fine pagina il codice HTML con i link
- puoi anche togliere le due righe di codice TWIG `{% if giuaReg is not null %} `  e `{% endif %}`  per rendere visibile il link predefinito all'app  GiuaReg
